src/libertem/common/udf.py

```diff
@@ -25,6 +25,12 @@ class TilingPreferences(TypedDict):
     total_size: Union[float, int]
 
 
+class UDFMethodEnum(Enum):
```

Comment:
Naming nit: how about just `UDFMethod`?

If we decide to make this a public interface, we may want to put these constants into the `UDF` class namespaces, too, like we have for array backend selection - this should reduce the number of imports needed for writing UDFs.

Reply:
OK by me, originally what I wrote but I was following the style of the other `Enums` in the file:

https://github.com/LiberTEM/LiberTEM/blob/675c98afd6101ea54d2c5e978d9d163ea93963b3/src/libertem/common/udf.py#L15-L20

Reply:
> OK by me, originally what I wrote but I was following the style of the other `Enums` in the file:
> 
> https://github.com/LiberTEM/LiberTEM/blob/675c98afd6101ea54d2c5e978d9d163ea93963b3/src/libertem/common/udf.py#L15-L20

I think they have the suffix because depth / size are either an integer, or a marker value from the `Enum`.